source/shared/cpp/ObjectModel/HostConfig.h

```diff
@@ -180,11 +218,71 @@ namespace AdaptiveSharedNamespace
         static MediaConfig Deserialize(const Json::Value& json, const MediaConfig& defaultValue);
     };
 
-    struct HostConfig
+    class HostConfig
     {
-        std::string fontFamily = "Calibri";
+    public:
+        HostConfig();
```

Comment:
I don't think you need this at the moment since we're not doing anything other than default initialization